packages/design-tokens/src/tokens/color.tokens.json

```diff
@@ -79,6 +79,9 @@
     "background-decorative-weak": {
       "value": "#FFF1DB"
     },
+    "background-outline-hover": {
```

Comment:
We don't need a background-outline token also?

Reply:
I believe that we are good because this will solve the contrast issue!